tests/ui/invalid-compile-flags/fuel.rs

```diff
@@ -0,0 +1,10 @@
+// revisions: incremental threads
+//
+// [threads] compile-flags: -Zfuel=a=1 -Zthreads=2
+// [threads] error-pattern:optimization fuel is incompatible with multiple threads
```

Comment:
if you're checking via a pattern here, might as well `dont-check-compiler-stderr` and remove the .stderrs?